EpiAware/docs/src/man/getting-started-julia.md

```diff
@@ -0,0 +1,199 @@
+# Getting Started with using Julia in Projects
+
+Julia is a programming language aimed at technical computing. This guide will help you set up Julia on your system according to our recommended best practices. This isn't a guide to learning the Julia programming language, but rather a guide to setting up your system to use Julia effectively in project workflows.
```

Comment:
It might be nice to add a "for whom" to this para. Is this guide for an epi modeler who knows python and R but not any Julia? Is it for an experienced Julia dev who will benefit from your opinionated setup when using this project? 

If you expect contributors to have read this doc and have their system setup appropriately, you might want to link it in the readme and contributing.md

Reply:
Good point. I'll have a quick think.

Reply:
I'll link in `README.md` if this goes through as a follow-up mini-PR